src/libCrypto/EthCrypto.cpp

```diff
@@ -415,7 +416,7 @@ bytes GetOriginalHash(TransactionCoreInfo const& info, uint64_t chainId) {
 
 // From a zilliqa TX, get the RLP that was sent to the node to create it
 std::string GetTransmittedRLP(TransactionCoreInfo const& info, uint64_t chainId,
-                              std::string signature) {
+                              std::string signature, uint64_t& recid) {
```

Comment:
make it such that we can extract the recovery ID when we get the transmitted RLP